src/Microsoft.AspNetCore.Blazor.Browser.JS/src/Platform/Mono/MonoDebugger.ts

```diff
@@ -0,0 +1,50 @@
+import { getAssemblyNameFromUrl } from '../DotNet';
+
+const currentBrowserIsChrome = (window as any).chrome
+  && navigator.userAgent.indexOf('Edge') < 0; // Edge pretends to be Chrome
+
+let hasReferencedPdbs = false;
+
+export function hasDebuggingEnabled() {
+  return hasReferencedPdbs && currentBrowserIsChrome;
+}
+
+export function attachDebuggerHotkey(loadAssemblyUrls: string[]) {
+  hasReferencedPdbs = loadAssemblyUrls
+    .some(url => /\.pdb$/.test(getAssemblyNameFromUrl(url, /* includeExtension */ true)));
```

Comment:
Should this just be a new function - `getFileNameFromUrl`? It really is a different thing...

Reply:
Yes, I agree. I'll change that.